### ISSUE TITLE
from source installation fails because `readme.md` is missing

### ISSUE THREAD

**Issue by boegel** [uburuntu/throttler#3]
I'm running into the following when using `pip install` using the source tarball for throttle 0.2.2 obtained from PyPI:

```
    Running command python setup.py egg_info
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/eb-pc17jo6j/pip-req-build-o1s1r0pd/setup.py", line 43, in <module>
        long_description=read('readme.md'),
      File "/tmp/eb-pc17jo6j/pip-req-build-o1s1r0pd/setup.py", line 10, in read
        with open(filename, encoding='utf-8') as file:
    FileNotFoundError: [Errno 2] No such file or directory: 'readme.md'
WARNING: Discarding file:///tmp/vsc40023/easybuild_build/snakemake/7.18.2/foss-2021b/throttler/throttler-1.2.1. Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

The problem is that `readme.md` is not included in the source tarball.

**Comment by boegel** [comment]
Likewise for `requirements-dev.txt`

**Comment by uburuntu** [comment]
Wow, I see, thanks for the report. Probably it's worth including these files to the package. Btw, what's the full command you tried to run? I can't reproduce, so far.

**Comment by chenrui333** [comment]
This is the full command that I used, `pip install -v --no-deps --no-binary :all: --ignore-installed throttler` (I can reproduce the same error)

**Comment by boegel** [comment]
> Wow, I see, thanks for the report. Probably it's worth including these files to the package. Btw, what's the full command you tried to run? I can't reproduce, so far.

After downloading and unpacking the source tarball from PyPI, from the unpacked sources, basically just:
```
pip install --prefix /tmp/$USER .
```

I'm actually using additional options for `pip install` ("`--verbose  --no-deps  --ignore-installed  --no-index  --no-build-isolation`"), but I'm pretty sure those don't matter.

**Comment by uburuntu** [comment]
**TL;DR**: Fixed with [v.1.2.2](https://github.com/uburuntu/throttler/releases/tag/v1.2.2).

Thanks folks for the report and details -- I fixed the issue by https://github.com/uburuntu/throttler/commit/912895b1231968412801ba883d49cc62174e8e89.

Used links for anyone who find the topic:
- https://stackoverflow.com/questions/23174516/read-readme-in-setup-py
- https://setuptools.pypa.io/en/latest/userguide/datafiles.html

That's weird that official documentation didn't highlight the issue in https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/. Any package that followed the guide will fail to installation from the source tarball 🤷🏻‍♂️
